test/cbmc/proofs/Makefile-template-defines

```diff
@@ -0,0 +1,18 @@
+
+# Absolute path to the root of the source tree.
+#
+SRCDIR ?= $(abspath $(PROOF_ROOT)/../../../../ota-for-aws-iot-embedded-sdk/)
```

Comment:
Can we remove an extra `../`? I mean we are going one extra level up to traverse down back again.
```suggestion
SRCDIR ?= $(abspath $(PROOF_ROOT)/../../../)
```

Reply:
Removed it in [00c584f](https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/245/commits/00c584ff56af2ffd29297de27a527a4687706687)